src/components/MainContentContainer.tsx

```diff
@@ -0,0 +1,71 @@
+import { useMediaQuery } from "react-responsive";
+import { Outlet } from "react-router-dom";
+import styled from "styled-components";
+
+import { useImageURI } from "globalstate/ImageURI";
+
+import { PanelContainer } from "components/PanelContainer";
+import { Button } from "components/Button";
+import { PreviewPanel } from "components/PreviewPanel";
```

Comment:
A-Z pls

Reply:
This file has been removed and adjusted into other components. But I HAD put them in Alpha before the move! 😸